src/SDK/Language.php

```diff
@@ -97,4 +97,17 @@ protected function toCamelCase($str): string
         $str = str_replace(" ", "", $str);
         return lcfirst($str);
     }
+
+    protected function toSnakeCase($str): string
+    {
+        $str = \preg_replace('/([a-z])([A-Z])/', '$1 $2', $str);
+        $str = \explode(' ', $str);
+        $str = \implode('_', $str);
+        return \strtolower($str);
+    }
+
+    protected function toUpperSnakeCase($str): string
+    {
+        return \strtoupper($this->toSnakeCase($str));
```

Comment:
do we really need this? Can we just not chain the twig filters directly

Reply:
We could chain them but I think it makes the templates/code easier to read if we just use one well-named filter